### PR TITLE
Adds a new maintenance tunnel

### PR DESCRIPTION
**Pull request by Yoshax**
The one on the right:
https://i.gyazo.com/be413fdb1707c6321930d78bd3c7f903.png

### CODE REVIEW

**Comment by EmperorJon** [comment]
That makes the wiring loop rather strange.

**Comment by Yoshax** [comment]
I'd rather not touch wiring.

**Comment by Hubblenaut** [comment]
I think you should add an air shutter there as well. Looks like a good place to separate maint shafts.

**Comment by Anewbe** [comment]
:+1:

**Comment by Yoshax** [comment]
Air shutters in my maintenance? Who are you, OSHA?

**Comment by SpadesNeil** [comment]
Too expensive? I don't understand.

**Comment by Yoshax** [comment]
Latter.

**Comment by Hubblenaut** [comment]
Seriously though, that spot on the level of the lower right hazard canister in the new maint shaft looks perfect for putting a door in - otherwise this is getting a bit too large.

**Comment by Yoshax** [comment]
Jeeee, fine. I'll put a door.

**Comment by Neerti** [comment]
Has a door been put in?

**Comment by Yoshax** [comment]
A door has been put in, as the commit says.

File: maps/polaris-1.dmm [hunk 1]
```diff
@@ -4649,6 +4652,7 @@
...
+"bLA" = (/obj/random/powercell,/obj/random/powercell,/obj/random/powercell,/obj/random/powercell,/obj/random/toolbox,/obj/effect/decal/cleanable/molten_item,/mob/living/simple_animal/hostile/mimic/crate,/turf/simulated/floor,/area/maintenance/medbay_fore)
```
Comment (SpadesNeil) [inline maps/polaris-1.dmm]: DAMN IT, YOSH!

**Pull request merged**